src/signals/settings/categories/Detail/index.js

```diff
@@ -96,7 +97,12 @@ const CategoryDetail = () => {
     (event) => {
       const formData = [...new FormData(event.target.form).entries()]
         // convert stringified boolean values to actual booleans
-        .map(([key, val]) => [key, key === 'is_active' ? val === 'true' : val])
+        .map(([key, val]) => [
+          key,
+          key === 'is_active' || key === 'is_public_accessible'
```

Comment:
Can you add a test that covers the branch?

Reply:
I did